common/util/web_controller/autoware_web_controller/package.xml

```diff
@@ -1,51 +1,18 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>autoware_web_controller</name>
   <version>0.1.0</version>
   <description>The autoware_web_controller package</description>
 
   <maintainer email="yukihiro.saito@tier4.jp">Yukihiro Saito</maintainer>
-  <license>Apache2</license>
+  <license>Apache License 2.0</license>
 
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <!-- Url tags are optional, but multiple are allowed, one per tag -->
-  <!-- Optional attribute type can be: website, bugtracker, or repository -->
-  <!-- Example: -->
-  <!-- <url type="website">http://wiki.ros.org/autoware_web_controller</url> -->
+  <exec_depend>rosbridge_server</exec_depend>
```

Comment:
Should some of the autoware message packages used in this web server be added here as dependencies since those will need to be built?

Reply:
I don't mind either way, but I was just applying comment from the related PR. https://github.com/tier4/Pilot.Auto/pull/104#discussion_r533314002

Reply:
Good catch. I added the dependencies